src/hcf-release/packages/generate-secrets/packaging

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e -x
+
+BIN_DIR=${BOSH_INSTALL_TARGET}/bin
+
+mkdir -p "${BIN_DIR}"
+
+cp -a "${BOSH_COMPILE_TARGET}/hcf-config/role-manifest.yml" "${BOSH_INSTALL_TARGET}"
+
+export GOROOT=$(readlink -nf /var/vcap/packages/golang1.7)
+export GOPATH=${BOSH_COMPILE_TARGET}
+export PATH=$GOROOT/bin:$PATH
+
+# BOSH copies files in without the `src` directory; put it back.
+# We can't use a symlink because go doesn't like those
+mkdir -p "${BOSH_COMPILE_TARGET}/src"
+mv "${BOSH_COMPILE_TARGET}/github.com" "${BOSH_COMPILE_TARGET}/src/github.com"
+
+# We currently have no way of getting a valid version number, as we have no git repo
+GOBIN="${BIN_DIR}" go install -ldflags="-X main.version=0.0.0" github.com/suse/scf-secret-generator
```

Comment:
No, the submodule dir is lowercase: 
```
$ ls -l ./src/hcf-release/src/github.com/suse
total 8
drwxrwxr-x 3 alexh alexh 4096 Nov 17 15:50 scf-secret-generator
drwxrwxr-x 7 alexh alexh 4096 Jun  7 11:00 testbrain
```

I'm conflicted about what to do here because the acceptance-tests-brain used lowercase, and I thought it would look strange to have both `SUSE` and `suse` dirs. And I'm not sure if that would break things on case-insensitive filesystems.

Reply:
On a on case-insensitive filesystem you cannot have both `suse` and `SUSE` as siblings in the same directory. If they are in different directories it should be fine. In the above example they would both be sub-dirs of `github.com`. That would break. IIRC a case-insensitive filesystem will report one or the other, and you will not know which, except by looking at the contents.